### PR TITLE
Update changelog ahead of 7.0.2 release

### PR DESCRIPTION
**Pull request by sruke**
Update changelog ahead of 7.0.2 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 See the [releases](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases) for details on bug fixes and added features.
 
+7.0.2
+======
+### Bug Fixes:
+- Resolved an issue where JsonWebToken properties would throw exceptions when the input string was 'null'. See PR[#2335](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2335) for details.
+
 7.0.1
```
Comment (FuPingFranco) [inline CHANGELOG.md:8]: Do we need this line?

**Review by FuPingFranco — APPROVED** [review]
Left a small comment. Otherwise, LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (sruke) [reply CHANGELOG.md:8]: An entry is added to the changelog whenever there is a bugfix or a new feature.

Comment (FuPingFranco) [reply CHANGELOG.md:8]: I see, Thanks Sruthi!

**Pull request merged**